benchmarks/kernels/weight_shapes.py

```diff
@@ -0,0 +1,43 @@
+# Weight Shapes are in the format
```

Comment:
This seems duplicated from [`vllm/benchmarks/cutlass_benchmarks/weight_shapes.py`](https://github.com/vllm-project/vllm/blob/e53dfd3eafd9bd5cc217c3796c7d1911a88ec893/benchmarks/cutlass_benchmarks/weight_shapes.py)

Reply:
ya it is, its just so the imports resolve, I tried moving `weight_shapes.py` up a level and using:

```
from ..weight_shapes import WEIGHT_SHAPES
```
in both `benchmarks/cutlass_benchmarks/w8a8_benchmarks.py` and `benchmarks/kernels/benchmark_machete.py` but this didn't work. Open to suggestions!

Reply:
the benchmarks directory is under:
`vllm/benchmarks`
not:
`vllm/vllm/benchmarks` 
unfortunately